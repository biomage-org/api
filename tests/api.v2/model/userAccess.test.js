jest.mock('../../../src/api.v2/helpers/generateBasicModelFunctions',
  () => jest.fn(() => ({ hasFakeBasicModelFunctions: true })));

const userAccess = require('../../../src/api.v2/model/userAccess');

describe('model/userAccess', () => {
  it('Returns the correct generateBasicModelFunctions', async () => {
    expect(userAccess).toEqual(
      expect.objectContaining({
        hasFakeBasicModelFunctions: true,
      }),
    );
  });
});
