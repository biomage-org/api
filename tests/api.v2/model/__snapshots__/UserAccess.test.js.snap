// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`model/userAccess getExperimentUsers work correctly 1`] = `
Array [
  Object {
    "email": "someUser@example.com",
    "name": "someUser-name",
    "role": "owner",
  },
]
`;
